src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
+    <Description>Azure HDInsight Management SDK Library</Description>
+    <AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
+    <VersionPrefix>3.0.0-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure HDInsight Management;HDInsight;HDInsight Management</PackageTags>    
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```

Comment:
@shefaliv the only supported target frameworks are net452 and netstandard1.4.
We do not support .NET Core as of now.